#!/usr/bin/python3

def raise_exception_msg(message=""):
	'''Function that raise a NameError exception with a message'''
	raise NameError(message)
